_editorial_policies/01_bylaws.md

```diff
@@ -0,0 +1,168 @@
+          votes of the Editorial Board and record minutes of meetings of the Editorial Board.
+
+   1. Selection process
+       1. Members of the Editorial Board will be elected by a two-thirds vote of the Editorial Board.
+         1. Managing editors and section lead editors will be elected by a two-thirds vote of the
+            entire Editorial Board by the other current members of the Editorial Board when there is a
+            vacancy in either capacity.
+         1. Associate editors will be elected based on:
+            1. Demonstrated scholarly expertise in computational molecular science, broadly defined.
+            1. Stated commitment to contribute to decision making and maintenance of LiveCoMS.
+            1. Candidates will be favored who have shown excellence in reviewing for LiveCoMS, 
+	    based on independent evaluations
+	    of the members of the Editorial Board, and who have a history of contributing manuscripts to LiveCoMS.
+            1. Diversity in career background, geographical perspectives, life experiences, and any other factors that are seen as beneficial to represent the broader community, provide fairness in decisions, and include a range of viewpoints.
+	    1. All members on the Editorial Board shall be elected in a fair, impartial, and confidential process without regard to race, nationality, religion, gender or gender identity, sexual orientation, or familial status.
+         1. Managing editors will be elected based on:
+            1. all criteria outlined for associate editors, plus demonstrated excellence serving as an associate editor.
+	 1. The Recording Secretary will be nominated by the Managing editors from among the associate editors.
+   1. Appointments Terms
+      1. Appointment for all positions are for three years, which may be extended for an unlimited 
+	 number of terms by a vote of two/thirds of the remaining Editorial Board. If the terms of more 
+         than one third of the Editorial Board end at the same time, all may remain in their positions for 
+         long enough for additional editors be appointed, or one month, which ever comes first. 
+      2. At the time the bylaws are adopted, the three managing editorial terms will be for two, three, and four years,
+         determined by mutual agreement or random chance if no agreement is reached.
+   1. Voting
+      1. All votes described in the bylaws are tallied based on the fraction
+         of the Editorial Board responding within one week written notice
+         via e-mail or other voting mechanism approved by the managing
+         editors. In exceptional circumstances advance notice may be given
+         of a rush decision, where the board may be given one week (or more)
+         notice that their votes will be needed within a shorter window,
+         such as a 12 hour window.
+
+1. Conflicts of interest
+   1. A 'conflict of interest' is defined as any of:
```

Comment:
Can we say "potential conflict of interest"? My sense is not that we are saying all of these things necessarily compromise people's judgment, but they might do so, and they might be seen by outsiders as potentially comprising judgment. It's thus not just that we want to avoid COIs but we want to avoid potential COIs.

Reply:
(LMK if you'd rather get a PR with feedback.)